docs/users/cli-usage.md

```diff
@@ -29,6 +29,109 @@ Available Commands:
+
+Below are quickstart instructions for each app type; Wordpress, Drupal 7, and Drupal 8.
+
+**Note:** If you do not have ddev already on your machine, please follow the [installation instructions](../index.md#installation) before beginning the quickstart tutorial. 
+### Wordpress Quickstart
+To get started using ddev with a Wordpress site, simply clone the site's repository and checkout its directory.
+```
+git clone https://github.com/user/worpress_site
+cd wordpress_site
+```
+From here we can start setting up ddev. Inside of your site's working directory, enter the command:
+```
+ddev config
+```
+
+_Note: ddev config will prompt you for a site name, docroot, and app type._
+
+After you've run `ddev config`, you're ready to start running your site. To start running ddev, simply enter:
+```
+ddev start
+``` 
+When running `ddev start` you should see output informing you that the site's environment is being started. If startup is successful, you'll see a message like the one below telling you where the site can be reached.
+```
+Successfully started wordpress_site
+Your application can be reached at: http://wordpress_site.ddev.local
+```
+Quickstart instructions regarding database imports, can be found under [Database Imports](#database-imports).
+### Drupal 7 Quickstart
+Beginning to use ddev with a Drupal 7 site is as simple as cloning the site's repository and checking out its directory.
+```
+git clone https://github.com/user/my_drupal7_site
+cd my_drupal7_site
+```
+Now to start working with ddev. Inside of your site's working directory, enter the following command:
+```
+ddev config
+```
+
+_Note: ddev config will prompt you for a site name, docroot, and app type._
+
+After you've run `ddev config` you're ready to start running your site. Run ddev using a simple:
+```
+ddev start
+``` 
+When running `ddev start` you should see output informing you that the site's environment is being started. If startup is successful, you'll see a message like the one below telling you where the site can be reached.
+```
+Successfully started my_drupal7_site
+Your application can be reached at: http://my_drupal7_site.ddev.local
+```
+Quickstart instructions regarding database imports, can be found under [Database Imports](#database-imports).
+### Drupal 8 Quickstart
+You can get started with Drupal 8 sites on ddev either by cloning a git repository or using a new or existing composer project.
+
+**Git Clone Example**
+```
+git clone https://github.com/user/my_drupal8_site
+cd my_drupal8_site
+```
+
+**Composer Setup Example**
+```
+composer create-project drupal-composer/drupal-project:8.x-dev my_drupal8_site --stability dev --no-interaction
+cd my_drupal8_site
+```
+_You can find more information on composer and how to use it [here](https://github.com/drupal-composer/drupal-project)._
```

Comment:
Leaving as a placeholder. It would be ideal if there was a drupal.org doc https://www.drupal.org/docs/develop/using-composer, but a quick search isn't turning anything useful up.

Reply:
There's an open discussion about bringing those things into drupal.org. I'd expect that could be done in about 5 years, but I've been wrong about that before :) Maybe it won't get done in 5 years.